spec/struct.dd

```diff
@@ -512,7 +512,7 @@ $(SECTION3 $(LEGACY_LNAME2 AssignOverload, assign-overload, Identity Assignment
         $(P where $(CODE opAssign) is a member function of S:)
 
         ---
-        ref S opAssign(S s)
+        ref S opAssign(ref S s)
```

Comment:
@9rnsr I think you meant assignment from **rvalue**. In that case, is `ref S opAssign()(auto ref S s)` a more correct representation?